keepsorted/line_group.go

```diff
@@ -69,13 +71,19 @@ func groupLines(lines []string, metadata blockMetadata) []lineGroup {
 				numUnmatchedStartDirectives--
 			}
 		}
+
+		if metadata.opts.Group && initialIndent == nil {
+			initialIndent = &indents[i]
+			log.Printf("initialIndent: %d", *initialIndent)
+		}
 	}
 	// finish an outstanding lineGroup and reset our state to prepare for a new lineGroup.
 	finishGroup := func() {
 		groups = append(groups, lineGroup{comment: slice(lines, commentRange), lines: slice(lines, lineRange)})
 		commentRange = indexRange{}
 		lineRange = indexRange{}
 		block = codeBlock{}
+		log.Printf("%#v", groups[len(groups)-1])
```

Comment:
Presumably this was meant to be deleted? It reads as debug logging, not permanent logging. Would be very verbose and not very useful.

Probably also the one a few lines above, too, as that runs practically every group, though at least that one has enough context that I could see it working (especially since it looks like Printf default to Trace level, which presumably isn't logged by default).

Reply:
Yep, `log.Printf` is trace level and doesn't show up unless you run keep-sorted more verbosely

Before this PR we were logging every group in keepsorted/block.go. I moved it here so that I could also see what `initialIndent` was set to in context to help track down some of these bugs. I've actually found this logging kinda useful for debugging user questions, so I'm inclined to keep it